governance/votes/Executive vote - April 18, 2024.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Template - [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist new address in the RWA015-A output conduit, USDP Input Conduit Management, Spark Proxy Spell - April 18, 2024
+summary:  Distribute AD compensation for March 2024 and AVC Member participation rewards for Q1 2024, transfer Dai to Aave as part of the SparkLend revenue sharing agreement, Whitelist a new Block Tower address in the RWA015-A output conduit convert USDP to Dai, and deposit to the Surplus Buffer, trigger a Spark proxy spell on Ethereum and Gnosis.
+date: 2024-04-18T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist new address in the RWA015-A output conduit, USDP Input Conduit Management, Spark Proxy Spell - April 18, 2024
```

Comment:
:yellow_circle: **moderate**

1. Capitalize Titles
2. BlockTower is one word
3. Minor format update

```suggestion
title: Template - [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist New Address in the RWA015-A Output Conduit, USDP Input Conduit Management, Spark Proxy Spell - April 18, 2024
summary:  Distribute AD compensation for March 2024 and AVC Member participation rewards for Q1 2024, transfer Dai to Aave as part of the SparkLend revenue sharing agreement, whitelist a new BlockTower address in the RWA015-A output conduit, convert USDP to Dai and deposit to the Surplus Buffer, trigger a Spark proxy spell on Ethereum and Gnosis.
date: 2024-04-18T00:00:00.000Z
address: "$spell_address"

---
# [Executive Proposal] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist New Address in the RWA015-A Output Conduit, USDP Input Conduit Management, Spark Proxy Spell - April 18, 2024
```